### PR TITLE
Move radio config into mbed_app.json

### PR DESCRIPTION
**Pull request by janjongboom**
@hasnainvirk Actually I'd suggest to move the target overrides into the Mbed OS targets. Because if you have a module with integrated LoRa radio the pins aren't gonna change anyway.

### CODE REVIEW

File: mbed_app.json [hunk 1]
```diff
@@ -1,5 +1,29 @@
 {
 	"config": {
+        "lora-radio": {
+            "help": "Which radio to use (options: SX1272,SX1276)",
+            "value": "SX1272"
+        },
+
+        "lora-spi-mosi":       { "value": "D11" },
```
Comment (janjongboom) [inline mbed_app.json]: These are the configs for sx1276 radio shield, i think they're the same for sx1272. So now if you have the shield just change the setting above and done.

**Comment by hasnainvirk** [comment]
@janjongboom Could you please open this PR for development branch ?

**Comment by janjongboom** [comment]
@hasnainvirk Done.

**Comment by janjongboom** [comment]
But development does not compile for me... You sure that's the right branch?

**Comment by hasnainvirk** [comment]
Hash at feature-branch is changed. That’s why it doesn’t compile.

**Comment by hasnainvirk** [comment]
@janjongboom Closing this as I took fixed your commit and took it in here https://github.com/ARMmbed/mbed-os-example-lorawan/pull/23

**Pull request closed without merging**